### PR TITLE
OD-626 [Feature] Localize date, time and number for notifications

### PR DESCRIPTION
**Pull request by OleksandrLavrynenko**
@romanyosyfiv

OD-626 https://weboo.atlassian.net/browse/OD-626

## Description
The date, time, and number of unread notifications that have been localized depend on your browser's language settings.

## Screenshots/screencasts
**ru-RU**
Screenshot_15

**en-EN**
Screenshot_14

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

File: js/build.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ Fliplet.Widget.instance('notification-inbox-1-0-0', function(data) {
   // Sample implementation to initialize the widget
   var inbox = new NotificationInbox(element, data);
 
+  $(element).translate();
```
Comment (tonytlwu) [inline js/build.js:12]: Does this component use any translation strings from translation.json? If not, then this line is unnecessary.

```suggestion
  $(element).translate();
```

Comment (OleksandrLavrynenko) [reply js/build.js:12]: @tonytlwu Isn't it necessary for TD and TN helpers in templanes?

Comment (tonytlwu) [reply js/build.js:12]: @OleksandrLavrynenko No, it isn't. `$(selector).translate()` is only required to translate the HTML elements with `data-translate` attributes.

Comment (OleksandrLavrynenko) [reply js/build.js:12]: Ok, I'll fix it

**Review by tonytlwu — APPROVED** [review]
@romanyosyfiv @inna-bieshulia This is ready for testing

Risk: Low
Affected: Dates and numbers in notification inbox and badges
Verify: Dates and numbers are correctly displayed & formatted

Squash and merge: Use PR title

**Pull request merged**